redhat-appstudio/remote-secret
# Only reconcile remote secrets when generation changes

### What does this PR do?
This should avoid infinite loop that sometimes happens when updating just the status of the resource.

### What issues does this PR fix or reference?
https://issues.redhat.com/browse/SVPI-664

### How to test this PR?
The operator should run normally. In environments where we see the endless reconciliation loop, the loop should no longer happen after upgrade.
